Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,67 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_you:
+    if not renpy.seen_label("mas_compliment_you_2"):
+        call mas_compliment_you_2
+    else:
+        call mas_compliment_you_3
+    return
+
+label mas_compliment_you_2:
+    m 1wubso "..."
+    m 1wubfb "[player]..."
+    
+    if mas_isMoniUpset(higher=True):
+        m 2hubfb "You have no idea what that means to me."
+        
+    elif mas_isMoniDis(True):    
+        m 1ekbfa "..."
+        m "Thank you."
+        
+    elif mas_isMoniBroken(True):
+        m 1rud "Hmmm... Thank you."
+        m 2eua "..."
+        $ mas_gainAffection(2,bypass=True)
+        
```

Comment:
Doesn't really matter, but I don't really like this whitespace line here and I think it would be easier to read if it wasn't there.

Reply:
It was meant as a spot for a neutral, slightly complimented Broken Moni expression. Not sure if it felt that way, though.

Reply:
??

I'm talking about this blank, empty line you have here. I think it makes it harder to read.

Reply:
You mean line 565, correct?

Reply:
Or 567?

Reply:
I'm talking about line 565. Line 567 separates the menu from the closing of the broken path, which might make it easier to read. 

The line above any comment is the line the comment is targeted at.

Reply:
It was meant as a space for an short expression change, unless there is a different method to keep the expression on-screen?

Reply:
I'm pretty sure blank lines and whitespace don't do anything. The only thing they're used for is separating different parts of code to make it easier for a person to read. 

I don't think there's any way to have a bit of delay and keep the expression longer before returning. You might be able to add waits into that ellipse to make the player stay on that dialogue line for longer though.

Reply:
Nah, It just gotta be there for a second or so. Just enough to notice.